src/components/nutrition/meal/edit-meal.tsx

```diff
@@ -215,10 +224,34 @@ function FoodRow({
               }
             />
           </TableCell>
-          <TableCell align="right">
-            <IconButton onClick={onClickRemove}>
-              <DeleteIcon />
-            </IconButton>
+          <TableCell className="w-1/12">
+            <Toolbar className="min-h-6 p-0" sx={{margin: "0 0 0 auto"}}>
+              {(food.accessLevel == "PRIVATE") ?
+                <IconButton title={"Nutrition facts"}
+                            onClick={ (event) => {
```

Comment:
Indentation is off, but I'll probably run [prettier](https://prettier.io/) to clean up some of the formatting and figure out how to add prettier to the pre-commit checks. So OK if you want to merge as-is.